pkgs/development/python-modules/fontParts/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, buildPythonPackage, fetchPypi
+, fonttools, lxml, fs, unicodedata2
+, defcon, fontPens, fontMath, booleanOperations
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "fontParts";
+  version = "0.9.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0hwzdppmrrw1xz49x36h6mcsrwya1f3zpqrc206y73j4pbn7fh0k";
+    extension = "zip";
+  };
+
+  propagatedBuildInputs = [
+    booleanOperations
+    fonttools
+    unicodedata2  # fonttools[unicode] extra
+    lxml          # fonttools[lxml] extra
+    fs            # fonttools[ufo] extra
+    defcon
+    fontPens      # defcon[pens] extra
+    fontMath
+  ];
+
+  checkPhase = "python Lib/fontParts/fontshell/test.py";
```

to support alternate interpreters
```suggestion
  checkPhase = ''
    ${python.interpreter} Lib/fontParts/fontshell/test.py
  '';
```

---

Thanks, fixed!